src/journal/bsod.c

```diff
@@ -0,0 +1,249 @@
+/* SPDX-License-Identifier: LPGL-2.1-or-later */
+
+#include <errno.h>
+#include <getopt.h>
+#include <linux/vt.h>
+#include <sys/ioctl.h>
+#include <stdio.h>
+
+#include "alloc-util.h"
+#include "build.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "io-util.h"
+#include "log.h"
+#include "main-func.h"
+#include "qrcode-util.h"
+#include "sd-id128.h"
+#include "sd-journal.h"
```

Comment:
move the two sd-* includes in a separate block above this one

Reply:
ohh.. I just saw this review and the one about the man page.
I'll add them to my next push sir.